README.md

```diff
@@ -1,148 +1,431 @@
+boolean or a variable to your liking.
+
+The following flags are available:
+* mscs-enabled - Enable or disable the world server.
+* mscs-version-type - Assign the version type (release or snapshot).
+* mscs-client-version - Assign the version of the client software.
+* mscs-client-jar - Assign the .jar file for the client software.
+* mscs-client-url - Assign the download URL for the client software.
+* mscs-client-location - Assign the location of the client .jar file.
+* mscs-server-version - Assign the version of the server software.
+* mscs-server-jar - Assign the .jar file for the server software.
+* mscs-server-url - Assign the download URL for the server software.
+* mscs-server-args - Assign the arguments to the server.
+* mscs-initial-memory - Assign the initial amount of memory for the server.
+* mscs-maximum-memory - Assign the maximum amount of memory for the server.
+* mscs-server-location - Assign the location of the server .jar file.
+* mscs-server-command - Assign the command to run for the server.
+
+The following variables may be used in some of the values of the above keys:
+* $JAVA - The Java virtual machine.
+* $CURRENT_VERSION - The current Mojang Minecraft release version.
+* $CLIENT_VERSION - The version of the client software.
+* $SERVER_VERSION - The version of the server software.
+* $SERVER_JAR - The .jar file to run for the server.
+* $SERVER_ARGS - The arguments to the server.
+* $INITIAL_MEMORY - The initial amount of memory for the server.
+* $MAXIMUM_MEMORY - The maximum amount of memory for the server.
+* $SERVER_LOCATION - The location of the server .jar file.
+
+#### Example key/value pairs
+
+    mscs-enabled=true
+    mscs-version-type=release
+    mscs-client-version=$CURRENT_VERSION
+    mscs-client-jar=$CLIENT_VERSION.jar
+    mscs-client-url=https://s3.amazonaws.com/Minecraft.Download/versions/$CLIENT_VERSION/$CLIENT_VERSION.jar
+    mscs-client-location=/opt/mscs/client/$CLIENT_VERSION
+    mscs-server-version=$CURRENT_VERSION
+    mscs-server-jar=minecraft_server.$SERVER_VERSION.jar
+    mscs-server-url=https://s3.amazonaws.com/Minecraft.Download/versions/$SERVER_VERSION/minecraft_server.$SERVER_VERSION.jar
+    mscs-server-args=nogui
+    mscs-initial-memory=128M
+    mscs-maximum-memory=2048M
+    mscs-server-location=/opt/mscs/server
+    mscs-server-command=$JAVA -Xms$INITIAL_MEMORY -Xmx$MAXIMUM_MEMORY -jar $SERVER_LOCATION/$SERVER_JAR $SERVER_ARGS
+    
+#### Enabling Forge, BungeeCord, and other server software
+Please visit the [wiki]
+(https://github.com/sandain/MinecraftServerControlScript/wiki/Server-Customization-Examples) 
+for additional information.
+
+### Adjusting global server settings
+In tandem with the `mscs.properties` file--which handles options for individual 
+worlds--
+is the `mscs.conf` or simply `mscs` file, which handles global server settings.
+
+**You must create this file**. By default, MSCS looks for this file in three 
```

Comment:
**You must create this file**. By default, MSCS looks for this file in three 

Might want to specify that this file only needs to exist if you want to alter the default settings.